testsuite/test_minio.py

```diff
@@ -0,0 +1,22 @@
+import filecmp
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from django.conf import settings
+
+import interface.backend.minio_api as storage
+
+
+def test_minio(client, live_server):
+    filepath = settings.BASE_DIR / 'testsuite' / 'test.zip'
+
+    with open(filepath, 'rb') as f:
+        storage.upload('myFile.zip', bytes(f.read()))
+
+    assert storage.exists('myFile.zip')
+
+    with TemporaryDirectory() as _tmp:
```

Comment:
Q: Why not use a temporary file instead of a temporary directory?

Reply:
The minio api wants a path where it will write the data. It can't use a file handle. [See this](https://github.com/vmck/acs-interface/blob/master/interface/backend/minio_api.py#L34-L41).